src/funtracks/import_export/import_from_geff.py

```diff
@@ -316,13 +316,6 @@ def import_from_geff(
     )
     # compute the 'area' attribute if needed
     if tracks.segmentation is not None and extra_features.get("area"):
```

Comment:
Could do this properly now based on the existing features and the features to compute